pkg/search/postgres/query/common.go

```diff
@@ -91,5 +95,26 @@ func MatchFieldQuery(dbField *walker.Field, value string, highlight bool, now ti
 	if !ok {
 		return nil, nil
 	}
-	return matchFieldQuery(dbField, field, value, highlight, now)
+
+	qualifiedColName := dbField.Schema.Table + "." + dbField.ColumnName
```

Comment:
`qualifiedColName := qualifyColumn(dbField.Schema.Table, dbField.ColumnName)`

Reply:
Unfortunately that function is in a different package.